tests/dataplicity/m2m/wsclient/test_channelfile.py

```diff
@@ -0,0 +1,33 @@
+from dataplicity.m2m.wsclient import ChannelFile
+
+
+class FakeClient(object):
+    """ The ChannelFile class calls
+        `self.client.channel_write` inside, therefore this simple class can
+        act as a client representative.
+    """
+    def __init__(self):
+        self.buffer = ''
```

Comment:
It should definitely be bytes. If you want to fix it on Python3, then it would by `sys.stdout.buffer.write` to bypass the encoding and write straight to the stdout.